pipenv/cli/command.py

```diff
@@ -454,7 +454,7 @@ def check(
     "--outdated", is_flag=True, default=False, help=u"List out-of-date dependencies."
 )
 @option("--dry-run", is_flag=True, default=None, help=u"List out-of-date dependencies.")
-@install_options
+@sync_options
```

Comment:
The package is passed to ```state.installstate.packages``` by ```package_arg```, and being used by https://github.com/pypa/pipenv/blob/50dcaca07a60483d83eaf2bca8d3a661fa765472/pipenv/cli/command.py#L482. Why it is replaced by ```sync_options```

Reply:
@jxltom, that was a mistake. However, state.installstate.packages are not used after this point.

Reply:
Oh it is closed. Forget about why I said. :p

Reply:
@frostming That's true. We should support it actually. 😄